docs/details/image.dox

```diff
@@ -908,6 +908,95 @@ double y_center = m01 / m00;
+
+\defgroup image_func_iterative_deconv Iterative Deconvolutions
+\ingroup imageflt_mat
+
+Iterative Deconvolution Algorithms
+
+The following table shows the iteration update equations of the respective deconvolution algorithms.
+
+<table>
+<tr><th>Algorithm</th><th>Update Equation</th></tr>
+<tr>
+    <td>VanCittert</td>
+    <td>
+        \f$ \hat{I}_{n} = \hat{I}_{n-1} + \alpha * (I - P \otimes \hat{I}_{n-1}) \f$
+    </td>
+</tr>
+<tr>
+    <td>Jansson-VanCittert</td>
+    <td>
+        \f$ \hat{I}_{n} = \hat{I}_{n-1} + \alpha * (1 - \frac{2*| \hat{I}_{n-1}-\frac{B}{2} |}{B}) * (I - P \otimes \hat{I}_{n-1}) \f$
+    </td>
+</tr>
+<tr>
+    <td>LandWeber</td>
+    <td>
+        \f$ \hat{I}_{n} = \hat{I}_{n-1} + \alpha * P^T \otimes (I - P \otimes \hat{I}_{n-1}) \f$
+    </td>
+</tr>
+</table>
+
+where
+    - \f$ I \f$ is the observed(input/blurred) image
+    - \f$ P \f$ is the point spread function
+    - \f$ P^T \f$ is the transpose of point spread function
+    - \f$ \hat{I}_{n} \f$ is the current iteration's updated image estimate
+    - \f$ \hat{I}_{n-1} \f$ is the previous iteration's image estimate
+    - \f$ \alpha \f$ is the relaxation factor
+    - \f$ \otimes \f$ indicates the convolution operator
+
+\note The type of output \ref af::array from deconvolution will be double if the input array type is
+double. Otherwise, it will be float in rest of the cases. Should the caller want to save the image to
+disk or require the values of output to be in a fixed range, that should be done by the caller
+explicitly.
+
+=======================================================================
+
+\defgroup image_func_inverse_deconv Inverse Deconvolution
+\ingroup imageflt_mat
+
+Inverse deconvolution is an linear algorithm i.e. they are non-iterative in nature and usually faster
+than iterative deconvolution algorithms.
+
+Depending on the values passed on to the enum \ref af_inverse_deconv_algo, different equations are
+used to compute the final result.
+
+#### Tikhonov's Deconvolution Method:
+
+The update equation for this algorithm is as follows:
+
+\f[
+\hat{I}_{\omega} = \frac{ I_{\omega} * P^{*}_{\omega} } { |P_{\omega}|^2 + \gamma }
+\f]
+
+where
+    - \f$ I_{\omega} \f$ is the observed(input/blurred) image in frequency domain
+    - \f$ P_{\omega} \f$ is the point spread function in frequency domain
+    - \f$ \gamma \f$ is a user defined regularization constant
+
+#### Weiner's Deconvolution Method:
+
+The update equation for this algorithm is as follows:
+
+\f[
+\hat{I}_{\omega} = \frac{ I_{\omega} * P^{*}_{\omega} } { |P_{\omega}|^2 + \frac{\gamma}{|I_{\omega}|^2 - \gamma} }
+\f]
+
+where
+    - \f$ I_{\omega} \f$ is the observed(input/blurred) image in frequency domain
+    - \f$ P_{\omega} \f$ is the point spread function in frequency domain
+    - \f$ \gamma \f$ is a user defined noise variance constant
+
+
+\note The type of output \ref af::array from deconvolution will be double if the input array type is
```

Comment:
What is the range of the float values. Is it 0-1 or 0-255?

Reply:
The algorithm is niether dependent on the range of the data nor alters the range of the data.